pydm_launcher/main.py

```diff
@@ -13,6 +12,16 @@ def main():
     logger.setLevel("INFO")
     handler.setLevel("INFO")
 
+    from pydm.utilities import is_ssh_session
+
+    if is_ssh_session():
+        logger.info('Using PyDM via SSH. Reverting to Software Rendering.')
```

Comment:
This would be useful as a utility function, in case a downstream library wants to use it and isn't using the pydm launcher.

Reply:
That is true. It was stupid of my part to make the `is_ssh_session` and not the whole thing as a utility function. I will push it.

Reply:
Addressed with https://github.com/slaclab/pydm/pull/661/commits/eb9f2def5caf56ee8699fff1c3fabdc67bafcf3c